src/client/pages/Offer/Checkout/InsuranceSummary.tsx

```diff
@@ -40,6 +40,38 @@ export const Row = styled.div`
   line-height: 1;
 `
 
+interface ErrorBoundaryProps {
+  children?: ReactNode
+  fallback?: ReactNode
+}
+
+interface ErrorBoundaryState {
+  hasError: boolean
+}
+
+class ErrorBoundary extends Component<ErrorBoundaryProps, ErrorBoundaryState> {
+  constructor(props: any) {
+    super(props)
+    this.state = { hasError: false }
+  }
+
+  static getDerivedStateFromError() {
```

Comment:
Nice! Didn't know about this lifecycle method. For reference: https://reactjs.org/docs/react-component.html#static-getderivedstatefromerror

Reply:
hehe yeah it's pretty obscure, in most cases I would use `react-error-boundary` but didn't want to add an external dependency